profiles/trajectory.properties

```diff
@@ -1,7 +1,7 @@
 fitMethod=HERMITE_CUBIC
 samples=HIGH
 dt=0.01
-maxVel=40
-maxAccel=40
+maxVel=80
+maxAccel=100
 maxJerk=2300
-wheelbase=25.091
+wheelbase=28.132472594682024
```

Comment:
Is this just an experimental value? If so, where'd it come from?

Reply:
It's closer to our "effective" width considering wheel scrub. We ran some testing involving turning the robot in a circle a lot and measuring the distance.

Reply:
oh hey it's that dude's suggestion